DigitalLearningSolutions.Data/Services/DelegateDownloadFileService.cs

```diff
@@ -1,30 +1,57 @@
 ﻿namespace DigitalLearningSolutions.Data.Services
 {
+    using System.Collections.Generic;
+    using System.Data;
     using System.IO;
     using System.Linq;
     using ClosedXML.Excel;
     using DigitalLearningSolutions.Data.DataServices;
     using DigitalLearningSolutions.Data.DataServices.UserDataService;
     using DigitalLearningSolutions.Data.Helpers;
+    using DigitalLearningSolutions.Data.Models.CustomPrompts;
+    using DigitalLearningSolutions.Data.Models.User;
 
     public interface IDelegateDownloadFileService
     {
         public byte[] GetDelegateDownloadFileForCentre(int centreId);

```

Comment:
Can we rename this to "GetDelegatesAndJobGroupsDownloadFileForCentre"? Currently it's a bit unclear what the difference is between the two methods.